com.unity.multiplayer.mlapi/Runtime/Spawning/NetworkSpawnManager.cs

```diff
@@ -635,30 +658,28 @@ internal void OnDespawnObject(NetworkObject networkObject, bool destroyGameObjec
                     });
                 }
 
-                var rpcQueueContainer = NetworkManager.RpcQueueContainer;
-                if (rpcQueueContainer != null)
+                var messageQueueContainer = NetworkManager.MessageQueueContainer;
+                if (messageQueueContainer != null)
                 {
                     if (networkObject != null)
                     {
                         // As long as we have any remaining clients, then notify of the object being destroy.
                         if (NetworkManager.ConnectedClientsList.Count > 0)
                         {
-                            var buffer = PooledNetworkBuffer.Get();
-                            using (var writer = PooledNetworkWriter.Get(buffer))
-                            {
-                                writer.WriteUInt64Packed(networkObject.NetworkObjectId);
 
-                                var queueItem = new RpcFrameQueueItem
+                            ulong[] clientIds = NetworkManager.ConnectedClientsIds;
+                            var context = messageQueueContainer.EnterInternalCommandContext(
```

Comment:
**Nit-Pick:**
Would like to see less new lines in the method calls with the ");" on the same line as the last parameter.
(this is in several other places but just adding one more reminder)